requirements/edx/base.txt

```diff
@@ -71,19 +71,19 @@ django-multi-email-field==0.5.1  # via edx-enterprise
 django-mysql==2.4.1
 git+https://github.com/edx/django-oauth-plus.git@b9b64a3ac24fd11f471763c88462bbf3c53e46e6#egg=django-oauth-plus==2.2.9.edx-4
 django-oauth-toolkit==1.1.3
-django-object-actions==1.1.2  # via edx-enterprise
+django-object-actions==2.0.0  # via edx-enterprise
```

Comment:
This release actually [dropped Python 2 support](https://github.com/crccheck/django-object-actions/blob/master/CHANGELOG.md).  It still seems to be working as far as tests are concerned, but we may want to hold it back until we've rolled out the Python 3 upgrade to prod next week.

Reply:
I just saw this too.  Working on adding a constraint now.